config/routes.rb

```diff
@@ -465,6 +466,7 @@
     :constraints => { :metadata => /(autoinstall\.scm|vm-profile\.scm|pkg-groups\.tar)/ }
   # get for end of build action
   get 'unattended/built/(:id(:format))', :controller => 'unattended', :action => 'built', :format => 'text'
+  post 'unattended/failed/(:id(:format))', :controller => 'unattended', :action => 'failed', :format => 'text'
```

Comment:
You probably want to expose the `built` end-point via post as well (at least the kickstart it updated to use that and it makes sense). We might need keep the `get 'unattended/built'` for compatibility reasons.

Reply:
why?

Reply:
I was referring to https://github.com/theforeman/community-templates/pull/461/files#diff-f889a99937d7b22813b17f56e48481a4R222

Reply:
Ah, sorry. Yeah, we should offer both GET and POST.

Reply:
Good idea, I am not going to propose GET for deprecation tho, there are so many scripts in the wild with GET already, that would be quite disturbing change. I will make a comment in routes there.